core/src/main/java/com/amplifyframework/devmenu/DeveloperMenu.java

```diff
@@ -33,6 +34,10 @@
  * developer menu.
  */
 public final class DeveloperMenu implements ShakeDetector.Listener {
+    /**
+     * Logger for the developer menu.
+     */
+    protected static final Logger LOG = Amplify.Logging.forNamespace("amplify:devmenu");
```

Comment:
Can it be just `private` (and thus, not need documentation?)
```suggestion
    private static final Logger LOG = Amplify.Logging.forNamespace("amplify:devmenu");
```

Reply:
I made the Logger protected in order to access it in DevMenuEnvironmentFragment. I was thinking it would be helpful to have a single logger instance for the developer menu that can be shared across the developer menu code.

Reply:
@eeatonaws That is probably a more efficient way to do it, yes. The rest of the code base right now uses multiple logger impls, even for the same tag, though. It would be sticking with consistency, anyway, to do the same.